src/config.js

```diff
@@ -1,47 +1,7 @@
 import fs from 'fs';
-import * as dotenv from 'dotenv';
-import * as dotenvExpand from 'dotenv-expand';
 import { isEmpty } from 'ramda';
-import { URL } from 'node:url';
 
-// Load .env files in a similar way to the bikehopper-ui frontend
-// .env.development.local, .env.development, .env, etc
```

Comment:
Why are you removing this? Isn't this going to break usage of `.env.development.local` files? Also unrelated to adding linting.